docs/cli/usage/containers.md

```diff
@@ -71,6 +71,70 @@ amplify push
 
 Once this completes your container will be built via an automated pipeline and deployed to Fargate Tasks on an ECS Cluster fronted by an Amazon API Gateway HTTP API using a direct Cloud Map integration to your VPC. If you selected *Yes* to protect your API with Authentication, an Amazon Cognito User Pool will be created with an Authorizer integration for that API.
 
+## Access existing AWS resource from container
+
+
+You can grant your Fargate Task access to your existing resources.  After running or `amplify add api`, the CLI generates a `custom-policies.json` under the folder `amplify/backend/api/<api-name>/custom-policies.json`. The file is where you can specify the resources and actions that grant Lambda Function or Fargate Task access.
```

Comment:
Same comments as above. Please take note and apply the same changes to the Fargate section.